shell/components/PromptRemove.vue

```diff
@@ -300,7 +300,12 @@ export default {
     },
     done() {
       if ( this.cachedDoneLocation && !isEmpty(this.cachedDoneLocation) ) {
-        this.currentRouter.push(this.cachedDoneLocation);
+        // Only push if the route will change otherwise we'll get a `NavigationDuplicated` exception
+        const resolvedRoute = this.currentRouter.resolve(this.cachedDoneLocation);
+
+        if (resolvedRoute.resolved.fullPath !== this.$route.fullPath) {
+          this.currentRouter.push(this.cachedDoneLocation);
+        }
```

Comment:
This was not a part of the original PR. This was needed as a part of the backport because vue2 treats pushing to the same location as an error where as vue3 doesn't. Without this change one of our e2e tests would fail.